src/FieldWarning/Assets/Units/UnitFactory.cs

```diff
@@ -82,7 +82,7 @@ private void AddMinimapIcon(GameObject unit, Color minimapColor)
             var minimapIcon = GameObject.Instantiate(Resources.Load<GameObject>("MiniMapIcon"));
             minimapIcon.GetComponent<SpriteRenderer>().color = minimapColor;
             minimapIcon.transform.parent = unit.transform;
-            minimapIcon.transform.localPosition = Vector3.zero;
+            minimapIcon.transform.localPosition = new Vector3(0,0.01f,0);
```

Comment:
I think adding a comment to the effect of 'we intentionally place the icon above terrain to prevent overlap/flickering' would be useful info for future programmers reading the code.